### PR TITLE
Adapt CI on RHEL-9 branch

### PR DESCRIPTION
**Pull request by jkonecny12**
Adapt rhel-9 branch for running tests. This patch set is doing this.

* adapt Makefile variables after branching
* remove packit support (I would like to enable it back when centos stream 9 is available)
* adapt Dockerfiles to run tests on rhel-9
* remove unnecessary stuff from Dockerfiles; like ELN support
* remove unnecassary workflow files after branching
* Adapt existing workflows to run tests on rhel-9

[Test run](https://github.com/jkonecny12/anaconda/pull/13).

### CODE REVIEW

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me, thank you!

How about porting back some of the changes?

**Review by poncovka — APPROVED** [review]
Looks good to me. Thanks!

File: dockerfile/anaconda-ci/Dockerfile [hunk 1]
```diff
@@ -41,14 +41,6 @@ RUN pip install \
   nose \
   rpmfluff
 
-# see https://github.com/martinpitt/anaconda/settings/actions/add-new-runner
-RUN mkdir actions-runner && cd actions-runner && \
-  URL_BASE=https://github.com/actions/runner/releases && \
-  LATEST_VER=$(basename $(curl -Ls -o /dev/null -w '%{url_effective}' $URL_BASE/latest)) && \
-  curl -L "$URL_BASE/download/$LATEST_VER/actions-runner-linux-x64-${LATEST_VER#v}.tar.gz" | tar xvz
```
[the remark this answers is not in the document]
Comment (jkonecny12) [reply dockerfile/anaconda-ci/Dockerfile:54]: Correct.

File: dockerfile/anaconda-rpm/Dockerfile [hunk 2]
```diff
@@ -16,6 +16,7 @@ RUN set -e; \
   rpm-build; \
   curl -L https://raw.githubusercontent.com/rhinstaller/anaconda/${git_branch}/anaconda.spec.in | sed 's/@PACKAGE_VERSION@/0/; s/@PACKAGE_RELEASE@/0/; s/%{__python3}/python3/' > /tmp/anaconda.spec; \
   rpmspec -q --buildrequires /tmp/anaconda.spec | xargs -d '\n' dnf install -y; \
+  dnf clean all; \
```
[the remark this answers is not in the document]
Comment (jkonecny12) [reply dockerfile/anaconda-rpm/Dockerfile:23]: Definitely!

[the remark this answers is not in the document]
Comment (jkonecny12) [reply dockerfile/anaconda-rpm/Dockerfile:23]: https://github.com/rhinstaller/anaconda/pull/3338

**Pull request merged**